README.md

```diff
@@ -1 +1,92 @@
-# fcc_tribute_page
+# Robot technology
+
+> The robot technology for who are willing to lean how to build or how to know the year it was discoverd.
```

Comment:
I think it would be more descriptive if we focus more on what this project does rather than the content. Can we rename the project name to something along the line of `"Robot Technology Page"`, this denotes that the project is about a web page.

As for the description, can you please highlight briefly what this page does and how it does it?